app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,12 @@
 class ApplicationMailer < ActionMailer::Base
   default from: 'no-reply@pjtraining.com'
   layout 'mailer'
+
+  def locale(user)
+    if user
+      I18n.locale = user.locale || I18n.default_locale
+    else
+      super
```

Comment:
Does `locale(user)` exist in the `ActionMailer::Base` 🤔

Reply:
I moved it from application_.controller